src/common/utxobased/network/BlockBookAPI.ts

```diff
@@ -27,20 +35,81 @@ export const infoMessage = (): PartialTask => {
   }
 }
 
+export const asIServerInfo = asObject({
```

Comment:
Say, what's with this "I" everywhere? as in "asIServerInfo", since when are we apple? ;)

Reply:
I completely agree. I have some branches which will rename these identifies to do away the meaningless I prefix. I see it as a "c-sharpy thing"; Microsoft-y.